ext/fileinfo/libmagic/funcs.c

```diff
@@ -60,15 +60,14 @@ protected int
 file_printf(struct magic_set *ms, const char *fmt, ...)
 {
 	va_list ap;
-	size_t len;
```

Comment:
If we touch libmagic, we should keep [libmagic.patch](https://github.com/php/php-src/blob/master/ext/fileinfo/libmagic.patch) up to date.

Reply:
Done